apps/mark-scan/frontend/src/app_cardless_voting.test.tsx

```diff
@@ -33,7 +33,7 @@ beforeEach(() => {
   apiMock = createApiMock();
   kiosk = fakeKiosk();
   window.kiosk = kiosk;
-  apiMock.setPaperHandlerState('waiting_for_ballot_data');
```

Comment:
These e2e tests misrepresented paper handler state, but it wasn't an issue before now because of the order we rendered `LoadPaperPage` in (ie. it used to be lower priority)